### DISCUSSION TITLE
Why Git commit contribute graph is not showing?

### DISCUSSION THREAD

**Discussion by kurodahiroyuki23** [community/community#33544] [Other Feature Feedback, Questions, & Ideas] [answered]
I would like to display my git commit contribute graph on my github profile. 
Do you know how to do it?

**Comment by radar-blip** [comment]
Its already showing.

**Reply by attractiveDev** [reply]
Hello, Nice to meet you.

**Reply by attractiveDev** [reply]
I am a senior blockchain developer with high skills and rich experience.

**Comment by strong-728** [comment, marked as answer]
Hello, Nice to meet you.
You can see the git commit contribut graph in the contribute seeting.
there are 2 options Private contributions and Activity overview.
So if you check the activity overview, you can see the graph.
![image](https://user-images.githubusercontent.com/92005974/191143974-6ceddeca-3416-4df9-be5f-334560116aca.png)

**Reply by kurodahiroyuki23** [reply]
Thanks. 0x-blockchain. I will keep you in mind.
